src/mcts/params.cc

```diff
@@ -49,6 +49,9 @@ const OptionId SearchParams::kMaxPrefetchBatchId{
     "When the engine cannot gather a large enough batch for immediate use, try "
     "to prefetch up to X positions which are likely to be useful soon, and put "
     "them into cache."};
+const OptionId SearchParams::kLogitQEnabledId{
+    "logit-q-enabled", "LogitQEnabled",
+    "Whether to sum Q+U in logit space or not."};
```

Comment:
"Sum Q+U in logit space." is enough, "whether to" and "or not" are not really informative. :)

Reply:
Removed those bits and added a bit of extra information.